refcards-generator/Readme.md

```diff
@@ -0,0 +1,29 @@
+# Refcards Generator 
+
+The goal of this project is to generate automatically HTML pages from the Refcards on Github thanks to NodeJs and Javascript.
+
+# First...
+
+To start this generator you'll need a computer with NodeJs and npm.
+
+# Prerequisite
+
+You can download NodeJs there :
+
+```
+https://nodejs.org
+```
+
+# Installation
+
+After installing NodeJs, you'll need to install the dependencies with npm :
+
+```
+npm install
+```
+
+Then you can start the generation using :
+
+```
+node index.js
```

Comment:
you are using a script into your `package.json` so change this documentation to use the npm script instead of directly node command ;)